index.html

```diff
@@ -9,13 +9,73 @@
     <![endif]-->
     <script src="js/vendor/d3.v3.min.js"></script>
     <link rel="stylesheet" href="css/charts.css">
+    <link rel="stylesheet" href="css/public_analytics.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
+
   <body>
     <div class="container">
-      <h1>GSA Public Analytics</h1>
-      <div id="charts">
-      </div>
+      <header id="usa_dot_gov_header">
+        <h1>USA.gov Header</h1>
+      </header>
+
+      <section id="title">
+        <h2>Digital Services Dashboard</h2>
+        <p>This page provides a summary of federal government website traffic data. It is updated every 5 minutes.</p>
+        <p><strong>Please note:</strong> We only have data for 34% of federal websites. <a href="#" title="More information about this data">Click here</a> to learn more.</p>
+      </section>
+
+      <section id="time_series">
+        <h3>Visitors Over Time</h3>
+        <div id="chart:visitors_over_time" class="chart_area">
```

Comment:
Can you put `:` in HTML IDs? I would think that'd conflict with the CSS `:` pseudoselector syntax (e.g. `:last-child`)